src/components/Recording.jsx

```diff
@@ -106,26 +102,32 @@ export default class Recording extends Component {
         };
 
         recognition.onerror = (event) => {
+            const errorLink = '//support.google.com/chrome/bin/answer.py?hl=en&amp;answer=1407892';
+            const chromeLink = 'chrome://settings/contentExceptions#media-stream-mic';
```

Comment:
I tested this out and it turned out we can't do a link to chrome settings like this. We can either remove the link altogether, or instead switch it with a webpage explaining how to enable the microphone. I'd like to do the second, but I just need to find a good webpage to link to like we have for `errorLink`.